test/game_test.rb

```diff
@@ -91,7 +91,7 @@ def test_it_has_a_total_score
   end
 
   def test_it_has_a_score_difference
-    assert_equal -1, @game_1.score_difference
+    assert_equal (-1), @game_1.score_difference
```

Comment:
Was this not working for you without the parens?

Reply:
Was working. Just got an "ambiguous syntax" message from my linter and thought it might add clarity.